.github/workflows/build-dev.yml

```diff
@@ -1,41 +1,8 @@
-name: Build, test and lint
+name: Bygg app og kjør tester
 on:
   workflow_dispatch:
   pull_request:
 
 jobs:
-  ktlint:
-    name: Ktlint
-    runs-on: ubuntu-latest
-    timeout-minutes: 2
-    permissions:
-      contents: read
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Set up JDK
-        uses: actions/setup-java@v4
-        with:
-          distribution: 'temurin'
-          java-version: '21'
-          cache: 'gradle'
-      - name: Kjør ktlint
-        run: ./gradlew spotlessCheck
   build:
-    name: Build and test
-    runs-on: ubuntu-latest
-    timeout-minutes: 7
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          persist-credentials: false
-          fetch-depth: 0 # brukes for å unngå feil i differ i sonar
-      - name: Set up JDK
-        uses: actions/setup-java@v4
-        with:
-          distribution: 'temurin'
-          java-version: '21'
-          cache: 'gradle'
-      - name: Test and build
-        run: ./gradlew build -PskipLint
+    uses: navikt/tilleggsstonader/.github/workflows/java-build-and-test.yml@main
```

Comment:
Må man ha med permissions?

Reply:
Trenger ikke det for denne jobben :) 

Som du ser så har den allere kjørt grønt selv uten permission, f.eks. [her](https://github.com/navikt/tilleggsstonader-sak/actions/runs/11104091084/job/30847446538)

Reply:
Ikke nødvendig :)